FirebaseTestProject/ViewModels/PostData.swift

```diff
@@ -25,4 +35,29 @@ import Foundation
             print(error)
         }
     }
+    
+    func loadFavorites() async {
+        do {
+            let favorites = try await PostService.getFavorites()
+            self.favorites = favorites
+        }
+        catch {
+            print(error)
+        }
+    }
+    
+    func remove(post: Post) {
+        // Will the task stop if we exit the function, prior to deletion?
+        Task {
+            try await PostService.delete(post: post)
```

Comment:
`try await PostService.delete(post: post)` does not compile:
> Incorrect argument label in call (have 'post:', expected 'favorite:')

Might you have defined a `delete(post:)` method and forgotten to include it here?

Reply:
Resolved by [commit 855d3615a5b3d4637e115a6ff19dc256c002f8fa](https://github.com/benstone1/InstaCademy/pull/13/commits/855d3615a5b3d4637e115a6ff19dc256c002f8fa)